File: Asocioacion/php/tercerEjercicio/Vuelo.php
<?php

class Vuelo{

    private string $puertoOrigen;
    private string $puertoDestino;
    private string $fechaVuelo;
    private string $horaVuelo;

    public function __construct(string $puertoOrigen, string $puertoDestino, string $fechaVuelo, string $horaVuelo){

        $this->puertoOrigen = $puertoOrigen;
        $this->puertoDestino = $puertoDestino;
        $this->fechaVuelo = $fechaVuelo;
        $this->horaVuelo = $horaVuelo;        

    }

    //Puerto de origen
    public function getPuertoOrigen(){
        return $this->puertoOrigen;
    }
    public function setPuertoOrigen($puertoOrigen){
        $this->puertoOrigen = $puertoOrigen;
    }

    //Puerto de destino
    public function getPuertoDestino(){
        return $this->puertoDestino;
    }
    public function setPuertoDestino($puertoDestino){
        $this->puertoDestino = $puertoDestino;
    }

    //Fecha de vuelo 
    public function getFechaVuelo(){
        return $this->fechaVuelo;
    }
    public function setFechaVuelo($fechaVuelo){
        $this->fechaVuelo = $fechaVuelo;        
    }

    //Hora de vuelo

    public function getHoraVuelo(){
        return $this->horaVuelo;
    }
    public function setHoraVuelo($horaVuelo){
        $this->horaVuelo = $horaVuelo;
    }

}

?>

File: Asocioacion/python/tercerEjercicio/Vuelo.py
class Vuelo:
    
    c = 0
    
    def __init__(self,puertoOrigen, puertoDestino, fechaVuelo, horaVuelo):
        self.puertoOrigen = puertoOrigen
        self.puertoDestino = puertoDestino
        self.fechaVuelo = fechaVuelo
        self.horaVuelo = horaVuelo
        self.__class__.c+=1
    
    #Puerto de origen
        
    def getPuertoOrigen(self):
        return self.puertoOrigen
    
    def setPuertoOrigen(self, puertoOrigen):
        self.puertoOrigen = puertoOrigen
        
    #Puerto de destino 
    
    def getPuertoDestino(self):
        return self.puertoDestino
    
    def setPuertoDestino(self,puertoDestino):
        self.puertoDestino = puertoDestino
        
    #Fecha del vuelo
    
    def getFechaVuelo(self):
        return self.fechaVuelo
    
    def setFechaVuelo(self, fechaVuelo):
        self.fechaVuelo = fechaVuelo
    
    #Hora del vuelo
    
    def getHoralVuelo(self):
        return self.horaVuelo
    
    def setHoraVuelo(self, horaVuelo):
        self.horaVuelo = horaVuelo